stl/inc/chrono

```diff
@@ -6546,6 +6583,12 @@ _NODISCARD bool _To_xtime_10_day_clamped(_CSTD xtime& _Xt, const _CHRONO duratio
     return _Clamped;
 }
 
+template <class _Rep, class _Period>
+_NODISCARD bool _To_xtime_10_day_clamped(_CSTD xtime& _Xt, const _CHRONO duration<_Rep, _Period>& _Rel_time) noexcept(
+    is_arithmetic_v<_Rep>) {
+    return _To_xtime_10_day_clamped_v2(_Xt, _Rel_time);
+}

```

Comment:
The "v2" version of this function is great, however I think that we actually should remove the originally-named function which calls the v2 version.  We follow this type of pattern when we are dealing with "v2" versions in separately-compiled code (because in that case we are required to continue to provide the old name, ideally with the old behavior), however in headers we follow an "add/remove" pattern.  You can see examples of this here: 
(as a struct)
https://github.com/microsoft/STL/blob/1866b848f0175c3361a916680a4318e7f0cc5482/stl/inc/execution#L2899
(as a function)
https://github.com/microsoft/STL/blob/1866b848f0175c3361a916680a4318e7f0cc5482/stl/inc/xmemory#L1152-L1163
Notice that in each of these cases, you won't be able to find a "v1" or original version of the struct/function - the preceding versions are simply removed.  We don't need to retain the old name, and in fact we shouldn't.  When we remove the old definition/name entirely, if the linker goes looking for the old name, it won't mistakenly find a function with new behavior -- it will use the definition in old obj files.  Removing the old name will ensure that we don't run into an ABI mismatch where we provide different behavior than is expected.